test/test_webservice.py

```diff
@@ -521,3 +521,13 @@ def test_set_retries_not_reached(self):
         self.assertTrue(TEMP_ERRORS_RETRIES > 1)
         self.assertEqual(request.mark_for_retry(), 1)
         self.assertFalse(request.max_retries_reached())
+
+    def test_queryargs(self):
+        request = WSRequest(
+            url='http://example.org/path?a=1',
+            method='GET',
+            handler=dummy_handler,
+            queryargs={'a': 2, 'b': 'x%20x', 'c': '1+2', 'd': '&', 'e': '?'},
+        )
+        # FIXME: check encoding
+        self.assertEqual(request.url().toString(), 'http://example.org/path?a=1&a=2&b=x x&c=1+2&d=%26&e=?')
```

Comment:
@phw: what do we expect here?

Reply:
I probably would have assumed the queryargs are considered totally unencoded and supposed to get submitted as is. Which means I'd expect it to encode the parameter `b` as `x%2520x`. Makes sense?

Reply:
Yes, I noticed `queryargs` are encoded sometimes: https://github.com/metabrainz/picard/blob/5fb2ad4017b16f9eb4bc609c34dba4627663ebe1/picard/webservice/api_helpers.py#L176

Reply:
Introduction of `unencoded_queryargs` solves that